packages/noya-state/src/selectors.ts

```diff
@@ -30,6 +30,15 @@ export const getCurrentPageIndex = (state: ApplicationState) => {
 
   return pageIndex;
 };
+
+export const getAllIndexPaths = (state: ApplicationState) => {
+  const pages = state.sketch.pages;
+  const indexPaths = pages.map((page) =>
+    Layers.findAllIndexPaths(page, () => true),
```

Comment:
See above, but, let's call this selector function `findLayerIndexPaths`, have it return `LayerIndexPath[]`, and pass a filter function into here instead of always returning true